packages/react-native-codegen/src/parsers/typescript/parser.js

```diff
@@ -183,12 +183,27 @@ class TypeScriptParser implements Parser {
 
   parseEnumMembersType(typeAnnotation: $FlowFixMe): NativeModuleEnumMemberType {
     const enumInitializer = typeAnnotation.members[0]?.initializer;
-    const enumMembersType: ?NativeModuleEnumMemberType =
-      !enumInitializer || enumInitializer.type === 'StringLiteral'
-        ? 'StringTypeAnnotation'
-        : enumInitializer.type === 'NumericLiteral'
-        ? 'NumberTypeAnnotation'
-        : null;
+    const enumInitializerType = enumInitializer?.type;
+
+    let enumMembersType: ?NativeModuleEnumMemberType = null;
+
+    switch (enumInitializerType) {
+      case undefined:
```

Comment:
Can you leave a comment explaining this case, because it's very very strange to see `undefined` as a case in a switch. It's almost worth pulling it out into its own if statement above the switch.

Reply:
Fixed, I've extracted this to a separate if statement above the switch